packages/wonder-blocks-dropdown/util/constants.js

```diff
@@ -41,8 +41,10 @@ export const defaultLabels = {
     filter: "Filter",
     noResults: "No results",
     selectNoneLabel: "Select none",
-    selectAllLabel: (numOptions: number) => `Select all (${numOptions})`,
+    selectAllLabel: (numOptions: number): string =>
+        `Select all (${numOptions})`,
     noneSelected: "0 items",
-    someSelected: (numSelectedValues: number) => `${numSelectedValues} items`,
+    someSelected: (numSelectedValues: number): string =>
+        `${numSelectedValues} items`,
     allSelected: "All items",
 };
```

Comment:
Why don't we need to create a type like `ColorTypes` for `defaultLabels`?

Reply:
Because flow didn't ask.

I think it's because they are primitive literals.